automation/publish.sh

```diff
@@ -11,6 +11,7 @@
 source automation/check-patch.setup.sh
 cd ${TMP_PROJECT_PATH}
 
+export PLATFORMS=all
```

Comment:
lets create a rehearse please before merging the PR
if you need help please let me know